src/Xunit.Xml.TestLogger/Xunit.Xml.TestLogger.csproj

```diff
@@ -21,4 +21,11 @@
     <PackageReference Include="System.ValueTuple" Version="4.3.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Stylecop.Analyzers" Version="1.1.118">
```

Comment:
We can update Stylecop version here: https://github.com/spekt/xunit.testlogger/blob/master/scripts/dependencies.props

Reply:
done